tardis/__init__.py

```diff
@@ -1,25 +1,16 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
+import sys
 
-# Packages may add whatever they like to this file, but
-# should keep this content at the top.
-# ----------------------------------------------------------------------------
-import os
-
-__all__ = ['__version__', 'test']
+__all__ = ['__version__', 'run_tardis', 'yaml_load']
```

Comment:
I am guessing you also want `run_tardis` and `yaml_load` in `__all__` since you specifically import them below? If not, I can remove.

Reply:
Hi @pllim, thank you for fixing my mistake. We don't use `yaml_load` as far as I know, but we do use `run_tardis`. So I think it should be good to remove the `yaml_load`.

Reply:
Okay I removed yaml_load. AFAIK there is no need to import it if it is not going to be exposed, so I removed the import as well. Thanks for the review!

Reply:
Thank you! This should be good to merge then, right?